test/Altinn.AccessManagement.Tests/Data/Json/InsertAccessManagementResource/input4.json

```diff
@@ -1,14 +1,14 @@
 [
   {
     "resourceregistryid": "string",
-    "resourcetype": "string2"
+    "resourcetype": "MaskinportenSchema"
   },
   {
     "resourceregistryid": "string2",
-    "resourcetype": "string"
+    "resourcetype": "MaskinportenSchema"
```

Comment:
Could diversify the tests with different valid resource type values. Does it support lowercase if that changes on the ResourceRegistry side?

Reply:
Yes. lowercase is also supported. I can add data with lowercase aswell